.changeset/silver-jokes-start.md

```
@@ -0,0 +1,5 @@
+---
+'@mastra/inngest': patch
+---
+
+Fixed step retryCount remaining 0 during retries under the Inngest execution engine
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)